provisioning/miniprov/hare_mp/main.py

```diff
@@ -144,6 +147,17 @@ def get_server_type(url: str) -> str:
         logging.error('Cannot get server type (%s)', error)
         return 'unknown'
 
+def is_stage_required(url: str) -> bool:
```

Comment:
The function name is very generic, however, we are using it for a specific purpose to identify if mkfs must be executed on this node or not.
I think, let's rename the function accordingly, `is_mkfs_required()`